archived/FS-1061-Allow-intrinsic-type-extensions-for-provided-types.md

```diff
@@ -171,7 +171,7 @@ The only drawback is the complexity of the addition, but the addition makes prov
 # Alternatives
 [alternatives]: #alternatives
 
-The alternative is to add features to the provided type by alternative means such as inheriting from the provided type or by augmenting another type which defers to the provided type internally.  These drawbacks are detailed in [Examples of current limitations that this RFC addresses](##examples-of-current-limitations-that-this-frc-addresses).
+The alternative is to add features to the provided type by alternative means such as inheriting from the provided type or by augmenting another type which defers to the provided type internally.  These drawbacks are detailed in [Examples of current limitations that this RFC addresses](#examples-of-current-limitations-that-this-rfc-addresses).
```

Comment:
I've removed a doubled `#` which I believe is incorrect for anchors as well as fixing the anchor ref...